python/lsst/verify/yamlutils.py

```diff
@@ -80,7 +80,8 @@ def load_all_ordered_yaml(stream, **kwargs):
     return yaml.load_all(stream, OrderedLoader)
 
 
-def _build_ordered_loader(Loader=yaml.Loader, object_pairs_hook=OrderedDict):
+def _build_ordered_loader(Loader=yaml.CSafeLoader,
+                          object_pairs_hook=OrderedDict):
```

Comment:
It's possible that this ordered dict code is no longer needed since python3.7 guarantees dicts are ordered.  The main problem with yaml is that on write it forces key sort (which can be disabled with `sort_keys=False`)

Reply:
Could be. I'm not sure how to check that a rewrite wouldn't have ill effects, and I agree that it's out of scope.